test/e2e_node/oomkiller_linux_test.go

```diff
@@ -120,18 +136,27 @@ func runOomKillerTest(f *framework.Framework, testCase testCase, kubeReservedMem
 			e2epod.NewPodClient(f).Create(context.TODO(), testCase.podSpec)
 		})
 
-		ginkgo.It("The containers terminated by OOM killer should have the reason set to OOMKilled", func() {
-			ginkgo.By("Waiting for the pod to be failed")
-			err := e2epod.WaitForPodTerminatedInNamespace(context.TODO(), f.ClientSet, testCase.podSpec.Name, "", f.Namespace.Name)
-			framework.ExpectNoError(err, "Failed waiting for pod to terminate, %s/%s", f.Namespace.Name, testCase.podSpec.Name)
+		if testCase.expectPodRunning {
+			ginkgo.It("The containers should not be OOMKilled", func() {
+				ginkgo.By("Waiting to ensure the pod is still running")
+				time.Sleep(10 * time.Second)
+				err := e2epod.WaitForPodsRunningReady(context.TODO(), f.ClientSet, testCase.podSpec.Name, 1, 30*time.Second)
```

Comment:
you can use ginkgo.Continuously or something like this.

Reply:
I didn't find `ginkgo.Continuously` method. May I ask you to give me a link to the documentation or something?

Reply:
sorry, `gomega.Consistently`